### PR TITLE
Added new Sysop commands, formatting

### PR DESCRIPTION
**Pull request by tuday2**
-Kick <user>
-Broadcast <message>
-Cleanup
-Cleanuped up formatting in globals

### CODE REVIEW

File: MBBSEmu/HostProcess/GlobalRoutines/SysopGlobal.cs [hunk 1]
```diff
@@ -198,5 +219,62 @@ private void RemoveKey(IReadOnlyList<string> commandSequence)
...
+            var userName = commandSequence[2];
+
+            if (_sessions.Values.Any(
+                s => string.Equals(s.Username, userName, StringComparison.CurrentCultureIgnoreCase)))
+            {
+                var channelToKick = _sessions.Values.FirstOrDefault(u =>
+                    u.Username.Equals(userName, StringComparison.InvariantCultureIgnoreCase));
+
+                //Not sure i need this second check -- IDE was worried?
+                if (channelToKick != null)
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/GlobalRoutines/SysopGlobal.cs:267]: `FirstOrDefault` will return null if no record is found.

You can invert this IF statement where if `channelToKick == null` display error and return. Would reduce nesting.

File: MBBSEmu/HostProcess/MbbsHost.cs [hunk 2]
```diff
@@ -1017,6 +1017,14 @@ private void PatchRelocation(MbbsModule module)
 
         private void ProcessNightlyCleanup()
         {
+            //Check to see if manual cleanup was called
+            var globalCleanup = (_globalCache.Get("SYSOPGLOBAL-CLEANUP") as bool?).GetValueOrDefault();
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/MbbsHost.cs:1021]: Do a `TryGet` on the cache, cleans up this statement

File: MBBSEmu/Program.cs [hunk 3]
```diff
@@ -199,6 +199,9 @@ static void Main(string[] args)
                 var globalCache = _serviceResolver.GetService<IGlobalCache>();
                 var fileHandler = _serviceResolver.GetService<IFileUtility>();
 
+                //Set default manual cleanup flag
+                globalCache.Set("SYSOPGLOBAL-CLEANUP", false);
```
Comment (enusbaum) [inline MBBSEmu/Program.cs:203]: If you change the code in MbbsHost to the TryGet<T> you don't need to set an initial value

**Review by enusbaum — CHANGES_REQUESTED** [review]

**Comment by tuday2** [comment]
Fixed -- ready for review

**Review by enusbaum — APPROVED** [review]

**Pull request merged**